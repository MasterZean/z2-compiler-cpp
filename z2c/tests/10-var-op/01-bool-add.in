{ val a = Bool{} + Bool{} }
{ val a = Bool{} + Small{} }
{ val a = Bool{} + Byte{} }
{ val a = Bool{} + Short{} }
{ val a = Bool{} + Word{} }
{ val a = Bool{} + Int{} }
{ val a = Bool{} + DWord{} }
{ val a = Bool{} + Long{} }
{ val a = Bool{} + QWord{} }
{ val a = Bool{} + Float{} }
{ val a = Bool{} + Double{} }
{ val a = Bool{} + Char{} }
{ val a = Bool{} + PtrSize{} }
