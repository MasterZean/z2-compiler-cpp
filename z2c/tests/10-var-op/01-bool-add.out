{
	int8 a = false + false;
}
{
	int8 a = false + 0;
}
{
	uint8 a = false + 0u;
}
{
	int16 a = false + 0;
}
{
	uint16 a = false + 0u;
}
{
	int32 a = false + 0;
}
{
	uint32 a = false + 0u;
}
{
	int64 a = false + 0ll;
}
{
	uint64 a = false + 0ull;
}
{
	float a = false + 0.0f;
}
{
	double a = false + 0.0;
}
{
	uint32 a = false + '\0';
}
{
	size_t a = false + 0u;
}
